server/templates/pages/src/index.html

```diff
@@ -32,7 +32,7 @@
           <!-- endbuild -->
         <![endif]-->
     </head>
-    <body>
+    <body{{#if flowId}} data-flow-id="{{flowId}}" data-flow-time="{{flowTime}}"{{/if}}>
```

Comment:
My mistake. I suggested the Mustache way. We use Handlebars on the backend
and Mustache on the front end. This predates my FxA tenure and I cannot
give you a reason.

On Mon, Mar 7, 2016 at 12:55 PM, Phil Booth <notifications@github.com>
wrote:

> In server/templates/pages/src/index.html
> <https://github.com/mozilla/fxa-content-server/pull/3562#discussion_r55198940>
> :
>
> > @@ -32,7 +32,7 @@
> >            <!-- endbuild -->
> >          <![endif]-->
> >      </head>
> > -    <body>
> > +    <body{{#if flowId}} data-flow-id="{{flowId}}" data-flow-time="{{flowTime}}"{{/if}}>
>
> I believe this should be:
>
> {{#flowId}} data-flow-id="{{flowId}}" data-flow-time="{{flowTime}}"{{/flowId}}
>
> Really? Everywhere else in this template uses #if, the handlebars docs
> say to do it that way and when I try your suggestion locally, it doesn't
> work. What am I missing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/fxa-content-server/pull/3562/files#r55198940>.
>